engine/src/test/scala/cromwell/engine/WorkflowDescriptorSpec.scala

```diff
@@ -5,8 +5,9 @@ import java.nio.file.{Files, Paths}
 import better.files._
 import com.typesafe.config.ConfigFactory
 import cromwell.CromwellTestkitSpec
+import cromwell.core.WorkflowId
 import cromwell.engine.backend.io._
-import cromwell.engine.backend.CallMetadata
+import cromwell.engine.backend.{WorkflowDescriptor, CallMetadata}
```

Comment:
Anytime a `main` class moves, its `test` should move too.

For example: once this spec is in the right package, this import will not be necessary.